src/pocketmine/tile/Beacon.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author Pocketmine Team
+ * @link http://www.pocketmine.net
+ * 
+ *
+*/
+
+ namespace pocketmine\tile;
+ 
+ use pocketmine\level\format\Chunk;
+ use pocketmine\nbt\tag\CompoundTag;
+ use pocketmine\nbt\tag\IntTag;
+ use pocketmine\nbt\tag\StringTag;
+ use pocketmine\nbt\tag\ByteTag;
+ use pocketmine\nbt\tag\FloatTag;
+ 
+ class Beacon extends Spawnable{
+ 
+ 	public function __construct(Chunk $chunk, CompoundTag $nbt){
```

Comment:
You're PRing to master, which doesn't have 1.0 changes yet. This should more correctly be FullChunk on the current master.